src/configs/environnements.js

```diff
@@ -4,27 +4,32 @@ dotenv.config();
 
 const port = process.env.PORT || 3000;
```

Comment:
This filename should be 'environments.js'

Reply:
I'm the one who wrote that name. Really sorry. Please @alainburindi change it

Reply:
it's done